src/executables/csql_launcher.c

```diff
@@ -283,6 +293,55 @@ main (int argc, char *argv[])
 	  csql_arg.skip_vacuum = true;
 	  break;
 
+	case CSQL_QUERY_OUTPUT_S:
+	  csql_arg.query_output = true;
+	  break;
+
+	case CSQL_QUERY_COLUMN_DELIMITER_S:
+	  {
+	    int len = strlen (optarg);
+
+	    if (len == 1)
+	      {
+		csql_arg.column_delimiter = optarg[0];
+	      }
+	    else if (len >= 2 && optarg[0] == '\\')
+	      {
+		if (optarg[1] == 't')
+		  {
+		    csql_arg.column_delimiter = '\t';
+		  }
+		else if (optarg[1] == 'n')
+		  {
+		    csql_arg.column_delimiter = '\n';
+		  }
+		else
+		  {
+		    csql_arg.column_delimiter = optarg[0];
+		  }
```

Comment:
_Isn't it_
csql_arg.column_delimiter = **optarg[1]**;

Reply:
thank you, your comment is right, so i will modify.